app/index.html

```diff
@@ -7,14 +7,15 @@
   <title>Microsoft identity platform</title>
   <link rel="SHORTCUT ICON" href="./favicon.svg" type="image/x-icon">
 
-  <!-- msal.min.js can be used in the place of msal.js; included msal.js to make debug easy -->
-  <script src="https://alcdn.msauth.net/browser/2.26.0/js/msal-browser.js"
-    integrity="sha384-fitpJWrpyl840mvd9nBFLGulqR4BJzvim0fzrXQKdsVh2AQzE4rTTJ0o5o+x+dRK"
+   <!-- msal.min.js can be used in the place of msal.js; included msal.js to make debug easy -->
+  <script src="https://alcdn.msauth.net/browser/2.29.0/js/msal-browser.js"
+    integrity="sha384-L8LyrNcolaRZ4U+N06atid1fo+kBo8hdlduw0yx+gXuACcdZjjquuGZTA5uMmUdS"
     crossorigin="anonymous"></script>
 
+
   <!-- To help ensure reliability, Microsoft provides a second CDN -->
   <script type="text/javascript">
-    if (typeof Msal === 'undefined') document.write(unescape("%3Cscript src='https://alcdn.msftauth.net/browser/2.26.0/js/msal-browser.js' type='text/javascript' crossorigin='anonymous' %3E%3C/script%3E"));
+    if (typeof Msal === 'undefined') document.write(unescape("%3Cscript src='https://alcdn.msauth.net/browser/2.29.0/js/msal-browser.js' type='text/javascript' crossorigin='anonymous' %3E%3C/script%3E"));
```

Comment:
```suggestion
    if (typeof Msal === 'undefined') document.write(unescape("%3Cscript src='https://alcdn.msftauth.net/browser/2.29.0/js/msal-browser.js' type='text/javascript' crossorigin='anonymous' %3E%3C/script%3E"));
```

msftauth is the domain for the alternative CDN.

Reply:
Actually, let's remove this entirely as injecting scripts with document.write is considered bad practice https://developer.chrome.com/blog/removing-document-write/

Can you also update the corresponding tutorial article afterwards? https://learn.microsoft.com/en-us/azure/active-directory/develop/tutorial-v2-javascript-auth-code

Reply:
Please also do the same for this PR https://github.com/Azure-Samples/ms-identity-b2c-javascript-spa/pull/21

Reply:
The [tutorial ](https://learn.microsoft.com/en-us/azure/active-directory/develop/tutorial-v2-javascript-auth-code)is using code injection it will update automatically when the code is merged.